src/components/modules/toolbar/index.ts

```diff
@@ -57,22 +57,6 @@ import { I18nInternalNS } from '../../i18n/namespace-internal';
  * @property {Element} nodes.defaultSettings   - Default Settings section of Settings Panel
  */
 export default class Toolbar extends Module {
-  /**
-   * HTML Elements used for Toolbar UI
-   */
-  public nodes: {[key: string]: HTMLElement} = {
```

Comment:
maybe we can create an interface that will represent the nodes' structure of the module?

Reply:
@gohabereg any suggestion?

Reply:
so it should be moved to the ModuleNodes